index.html

```diff
@@ -8,49 +9,254 @@
+
+                <ul class="language">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <div class="button"><a href="#" class="see-btndark">See Project</a>
+                </div>
+            </div>
+        </div>
+        <div class="card">
+
+            <div class="right">
+                <img class="snapshot" src="images/S2.png" alt="snapshot">
+                <h2>Tonic</h2>
+                <ul class="detail">
+                    <li class="main">CANOPY</li>
+                    <li class="main2">Back End Dev </li>
+                    <li class="main2">2015</li>
+                </ul>
+
+                <div class="paragraph">
+                    <p class="short.descr">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.
+                    </p>
+                </div>
+
+                <ul class="language">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <div class="button"><a href="#" class="see-btndark">See Project</a>
+                </div>
+            </div>
+        </div>
+        <div class="card">
+
+            <div class="right">
+                <img class="snapshot" src="images/S3.png" alt="snapshot">
+                <div class="card-text">
+                    <h2>Multi-Post Stories</h2>
+                    <ul class="detail">
+                        <li class="main">CANOPY</li>
+                        <li class="main2">Back End Dev </li>
+                        <li class="main2">2015</li>
+                    </ul>
+                </div>
+                <div class="paragraph">
+                    <p class="short.descr">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.
+                    </p>
+                </div>
+
+                <ul class="language">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <div class="button"><a href="#" class="see-btndark">See Project</a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about_section">
+        <div class="about_me_title">
+            <h3 class="abt-heading">About<br>
+                Myself</h3>
+
+            <p class="abt-desc">
+                Hello I’m a software developer! I can help<br>
+                you build a product , feature or website <br>
+                Look through some of my work and<br>
+                experience! If you like what you see and<br>
+                have a project you need coded, don’t<br>
+                hestiate to contact me.
+            </p>
+        </div>
+        <div class="social-media">
+            <a href="#" class="contact">LET'S CONNECT </a>
```

Comment:
- [ ] Please I highly recommend using `h2` to `h6` tags instead of an `a` tag to follow the semantic rules of the web page. 😃